internal/checks/errorlatency.go

```diff
@@ -48,8 +48,15 @@ func (e *ErrorStats) RecordResponse(isError bool) {
 }
 
 func (e *ErrorStats) IsOpen() bool {
-	// TODO(polsar): Dedup with LatencyStats.IsOpen.
-	return e.circuitBreaker.IsOpen() || e.circuitBreaker.IsHalfOpen()
+	// TODO(polsar): We should be able to check `e.circuitBreaker.IsOpen()`,
+	//  but it appears to remain open forever, regardless of the configured delay.
+	//  We also must reset the circuit breaker manually if it is not supposed to be open.
+	isOpen := e.circuitBreaker.RemainingDelay() > 0
```

Comment:
weird, is this a bug in the library? The docs state that the breaker should change from `closed` to `halfopen` (e.g. `e.circuitBreaker.IsHalfOpen()`) after [a delay](https://failsafe.dev/circuit-breaker/) 👀 

Maybe we need to set `WithSuccessThreshold` when building the circuit breaker so that it knows that it can switch back to `HalfOpen`/`Open`? From the docs: `If the trial executions meet a success threshold, the breaker is closed again and executions will proceed as normal, otherwise it’s re-opened.` We don't have a success threshold defined, so maybe we need to do that?